common/src/main/java/io/netty/util/AbstractReferenceCounted.java

```diff
@@ -42,7 +42,7 @@ protected long unsafeOffset() {
 
     // Value might not equal "real" reference count, all access should be via the updater
     @SuppressWarnings("unused")
-    private volatile int refCnt = updater.initialValue();
+    private final int refCnt = updater.initialValue();
```

Comment:
Same as the above comment, this is updated by the updater.

Reply:
Done